logstash-core/spec/api/lib/api/plugins_spec.rb

```diff
@@ -0,0 +1,45 @@
+# encoding: utf-8
+require_relative "../../spec_helper"
+require "sinatra"
+require "app/modules/plugins"
+require "logstash/json"
+
+describe LogStash::Api::Plugins do
+
+  include Rack::Test::Methods
+
+  def app()
+    described_class
+  end
+
+  before(:all) do
+    get "/"
+  end
+
+  let(:payload) { JSON.parse(last_response.body) }
+
+  it "respond to plugins resource" do
+    expect(last_response).to be_ok
+  end
+
+  it "return valid json content type" do
+    expect(last_response.content_type).to eq("application/json")
+  end
+
+  it "return the expected schema" do
+    expect(payload.keys).to include("plugins", "total")
+    payload["plugins"].each do |plugin|
+      expect(plugin.keys).to include("name", "version")
```

Comment:
Sorry to be picky, I think we should make this test a bit more robust and check to make sure the value isn't nil. 

TIL that the `include` matcher only check for the presence of they, maybe it was obvious to everyone that it was calling `include?` on the hash, but for me it did not ring a bell until this morning.

This code change make the test pass.
```diff
diff --git a/logstash-core/lib/logstash/api/lib/app/commands/system/plugins_command.rb b/logstash-core/lib/logstash/api/lib/app/commands/system/plugins_command.rb
index 5b1b045..c1ffb77 100644
--- a/logstash-core/lib/logstash/api/lib/app/commands/system/plugins_command.rb
+++ b/logstash-core/lib/logstash/api/lib/app/commands/system/plugins_command.rb
@@ -11,7 +11,7 @@ class LogStash::Api::PluginsCommand < LogStash::Api::Command

   def plugins
     @plugins ||= find_plugins_gem_specs.map do |spec|
-      { :name => spec.name, :version => spec.version.to_s }
+      { :name => nil, :version => nil }
     end
   end
```